scripts/update_vega_version.sh

```diff
@@ -8,7 +8,7 @@ VEGA_SIM_NETWORKS_TAG=master" > .env
 
 make
 make proto
-pytest -s -v --log-cli-level INFO -m integration
+# pytest -s -v --log-cli-level INFO -m integration
```

Comment:
Is this meant to be commented out still?

Reply:
It is not

Reply:
:+1: